pkg/errcode/errcode.go

```diff
@@ -269,6 +269,9 @@ const (
 
 	// ErrSDSCertMismatch indicates the indentity obtained from the SDSCert request does not match the identity of the proxy
 	ErrSDSCertMismatch
+
+	// ErrTLSParameters indicates unsupported TLS parameters configuration
+	ErrTLSParameters
```

Comment:
Looks like this is no longer referenced anywhere, so I'd be in favor of removing it.